core/discord_bot.py

```diff
@@ -762,28 +806,23 @@ async def send_discord_message(self, message):
                 )
                 await self.debug_webhook.send("bot-guild-muted")
                 await self.send_message(embed=embed)
-    
-            # /g list command response
-            elif "Total Members:" in message:
-                message = re.split("--", message)
-                embed = ""
-                length = len(message)
-                for i in range(length):
-                    if i == 0:
-                        pass
-                    elif i % 2 == 0:
-                        ii = i - 1
-                        embed += "**" + message[ii] + "** " + message[i]
-                embed = Embed(description=embed.replace("_", "\\_"), colour=0x1ABC9C)
-                await self.send_debug_message("Sending player list message")
-                await self.send_message(embed=embed)
-    
-            # Everything else is sent as a normal message
+
             else:
                 if message.strip() == "":
                     return
-                await self.send_debug_message(f"Normal message: `{ message }`")
-                embed = Embed(colour=0x1ABC9C).set_author(name=message)
-                await self.send_message(embed=embed)
+                parser = GuildMessageParser(message)
+                embeds = parser.parse()
+                if isinstance(embeds, list):
+                    await self.send_debug_message("Sending guild command response")
+                    for embed in embeds:
+                        await self.send_message(embed=embed)
+                elif isinstance(embeds, dict):
+                    await self.send_debug_message("Sending guild command response")
+                    await self.send_message(embed=embeds["embed"])
+                    await self.send_image(image_file=embeds["file"])
```

Comment:
add this to the `file` kwarg in the above line

Reply:
done